naga/src/back/spv/block.rs

```diff
@@ -24,7 +24,7 @@ fn get_dimension(type_inner: &crate::TypeInner) -> Dimension {
 /// types are simply the direct SPIR-V analog of the Naga IR's. But in some
 /// cases, the Naga IR and SPIR-V types need to diverge.
 ///
-/// This enum specifies how [`BlockContext::write_expression_pointer`] should
+/// This enum specifies how [`BlockContext::write_access_chain`] should
```

Comment:
@jimblandy: Heads up, docs weren't updated. I just added a fix via force-push to 
[7c7da5b](https://github.com/gfx-rs/wgpu/compare/4db573d7dfadcf6eeaa41b0384358cdb477771d1..7c7da5b74ba429065816608292489f51acbb1eb1).